Assets/HoloToolkit/Input/Scripts/InputSources/InteractionSourceInfo.cs

```diff
@@ -5,9 +5,16 @@ namespace HoloToolkit.Unity.InputModule
 {
     public enum InteractionSourceInfo
     {
+        // UnityEngine.XR.WSA.Input.InteractionSourceKind
         Other = 0,
         Hand = 1,
         Voice = 2,
         Controller = 3,
+        // Other
+        Head,
+        Mouse,
+        Gamepad,
+        Keyboard,
+        TouchScreen,
```

Comment:
If we're defining the enum values we should probably use the flags attribute and make the assignments bit operators

Reply:
Also, it seems like some of these changes are beyond the scope of just multi pointers

Reply:
Yeah we can drop this stuff.